cfr.yaml

```diff
@@ -130,4 +130,4 @@
         cap: 1
   # Not default browser and user dismissed the default browser prompt
   targeting:
-    "firefoxVersion >= 87 && !isDefaultBrowser && !'browser.shell.checkDefaultBrowser'|preferenceValue && isMajorUpgrade != true && platformName != 'linux' && ((currentDate|date - profileAgeCreated) / 604800000) >= 5 && !activeNotifications"
+    "(firefoxVersion >= 87 || (firefoxVersion >= 90 && source == 'startup')) && !isDefaultBrowser && !'browser.shell.checkDefaultBrowser'|preferenceValue && isMajorUpgrade != true && platformName != 'linux' && ((currentDate|date - profileAgeCreated) / 604800000) >= 5 && !activeNotifications"
```

Comment:
Shouldn't this be `((firefoxVersion >= 87 && firefox < 90) || (firefoxVersion >= 90 && source == 'startup')) && ..`? Otherwise, this block will always be evaluated to be `true` for any Firefox version >= 87.

Reply:
Yeah that's ok in firefox [87, 90) we don't have the double trigger happening in those versions.

Reply:
The bug is for firefox version 90+, even if the source is not startup, the condition passes `version >= 87`.

Alternatively, would `version >= 87 && source != 'newtab' && !isDefault…` work?

Reply:
for firefox 87 the source is “newtab” so maybe something like:
“(firefoxVersion >= 87 && firefoxVersion < 90) || (firefoxVersion >= 90 && source == ‘newtab’)”

Reply:
We don’t care about source before 90 and after 90 we want to ignore newtab and only show on startup.

Reply:
That looks good to me. One little comment in the YAML file would be helpful for future references.